paper-toolbar.html

```diff
@@ -208,7 +212,7 @@
         text-overflow: ellipsis;
         font-size: 20px;
         font-weight: 400;
-        line-height: 1;
+        line-height: var(--paper-toolbar-height, 64px);
```

Comment:
`line-height: normal;` should do it actually - I had to look more into this recently for PolymerElements/paper-checkbox#106. :)